uisdk/src/main/res/values-night/colors.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
Since the color palette has been updated, shall we mark the old colors as derecated to align these warnings with iOS? At least the ones that have a direct hexstring and are not pointing to another kh_uisdk color. 

This deprecation shouldn't be applied to the new colors, those should be just updated directly.

Reply:
@dianakarh think all colors are used, no unused resources warnings